### ISSUE TITLE
Crash on terrain generation with this and Simply Skills together

### ISSUE THREAD

**Issue by FedUpWizard** [ZsoltMolnarrr/Rolling#34]
**Minecraft version** - 1.20.1
**Mod loader** - Quilt 0.21.1
**Mod version** - 1.3.0
**I am using the latest version available** - Yes

**Describe the bug**
The game crashes on finishing terrain generation, this only started after installing Simply Skills: https://www.curseforge.com/minecraft/mc-mods/simply-skills. Already reported there and was told to report it here since the log isn't Simply skills related.

**To Reproduce**
1. Start a world, wait on terrain generation.
2. Crash

**Expected behavior**
No crashes on generating terrain

**Screenshots or video recordings**
Will add screenshots if it becomes necessary

**Additional context**
[crash-2023-11-10_21.22.03-client.txt](https://github.com/ZsoltMolnarrr/CombatRoll/files/13329468/crash-2023-11-10_21.22.03-client.txt)

**Comment by ZsoltMolnarrr** [comment]
Installing Combat Roll and Simply Skills only does not reproduce this issue.
I need the absolute minimum number of mods that can reproduce this problem consistently.

**Comment by FedUpWizard** [comment]
Wow ok interesting I will have a look and will get back to you when I have narrowed it down.

**Comment by FedUpWizard** [comment]
[crash-2023-11-14_09.33.07-client.txt](https://github.com/ZsoltMolnarrr/CombatRoll/files/13341011/crash-2023-11-14_09.33.07-client.txt)
Ok I believe that is the minimum mods required and it does produce the crash. You may have missed the requirements unsure at the bottom of simply skills and I believe spell engine requires trinkets.

**Comment by FedUpWizard** [comment]
![image](https://github.com/ZsoltMolnarrr/CombatRoll/assets/109249618/72ae8555-a098-474b-ae75-0191bfec0a0f)
https://www.curseforge.com/minecraft/mc-mods/simply-skills
https://www.curseforge.com/minecraft/mc-mods/cloth-config/files?page=1&pageSize=20
https://www.curseforge.com/minecraft/mc-mods/spell-engine
https://www.curseforge.com/minecraft/mc-mods/qsl/files?page=1&pageSize=20
https://www.curseforge.com/minecraft/mc-mods/trinkets
https://www.curseforge.com/minecraft/mc-mods/spell-power
https://www.curseforge.com/minecraft/mc-mods/puffish-skills
https://www.curseforge.com/minecraft/mc-mods/combat-roll
https://www.curseforge.com/minecraft/mc-mods/playeranimator
https://www.curseforge.com/minecraft/mc-mods/cloth-config/files?page=1&pageSize=20
https://quiltmc.org/en/install/client/

Here so you dont have to go searching for them, I believe it is all of them :) 
I am not in a rush take your time, and have a grand day or night ;)

**Comment by aninusmuffin** [comment]
This seems to only happen when QKL/ Fabric Language Kotlin is installed.

### Logs:

- Simply Skills + Combat Roll (No QKL): https://mclo.gs/TeeWt5A
- Simply Skills + Combat Roll + QKL: https://mclo.gs/A9220T1 (**Crashed**)
- Simply Skills + Combat Roll + Fabric Language Kotlin : https://mclo.gs/kFzw0U0 (**Crashed**)
- Combat Roll + QKL : https://mclo.gs/2I76WEa (NO Simply Skills!)

### Error message:

```
[11:25:48] [Render thread/ERROR]: Unreported exception thrown!
java.lang.IllegalArgumentException: Can't find attribute combatroll:count
	at net.minecraft.class_5132.method_26865(class_5132.java:22) ~[transformed-mod-minecraft.i0:0/:?]
	at net.minecraft.class_5132.method_26862(class_5132.java:28) ~[transformed-mod-minecraft.i0:0/:?]
	at net.minecraft.class_5131.method_26852(class_5131.java:76) ~[transformed-mod-minecraft.i0:0/:?]
	at net.minecraft.class_1309.method_26825(class_1309.java:1904) ~[transformed-mod-minecraft.i0:0/:?]
	at net.combatroll.api.EntityAttributes_CombatRoll.getAttributeValue(EntityAttributes_CombatRoll.java:52) ~[transformed-mod-combatroll.i0:0/:?]
	at net.combatroll.client.RollManager.tick(RollManager.java:55) ~[transformed-mod-combatroll.i0:0/:?]
	at net.minecraft.class_310.tryRolling(class_310.java:3560) ~[transformed-mod-minecraft.i0:0/:?]
	at net.minecraft.class_310.handler$zpa000$combatroll$tick_TAIL(class_310.java:3552) ~[transformed-mod-minecraft.i0:0/:?]
	at net.minecraft.class_310.method_1574(class_310.java:1957) ~[transformed-mod-minecraft.i0:0/:?]
	at net.minecraft.class_310.method_1523(class_310.java:1181) ~[transformed-mod-minecraft.i0:0/:?]
	at net.minecraft.class_310.method_1514(class_310.java:802) ~[transformed-mod-minecraft.i0:0/:?]
	at net.minecraft.client.main.Main.main(Main.java:250) ~[minecraft-1.20.1-client.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:551) ~[quilt-loader-0.21.2.jar:?]
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:84) ~[quilt-loader-0.21.2.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:28) ~[quilt-loader-0.21.2.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
```

**Comment by ZsoltMolnarrr** [comment]
I would like to remind every that Quilt is not officially supported, hence issues submitted with it have very low priority.
If it works, that is great, otherwise there is no development capacity to do anything with that loader.

**Comment by FedUpWizard** [comment]
> This seems to only happen when QKL/ Fabric Language Kotlin is installed.
> ### Logs:
> 
>     * Simply Skills + Combat Roll (No QKL): https://mclo.gs/TeeWt5A
> 
>     * Simply Skills + Combat Roll + QKL: https://mclo.gs/A9220T1 (**Crashed**)
> 
>     * Simply Skills + Combat Roll + Fabric Language Kotlin : https://mclo.gs/kFzw0U0 (**Crashed**)
> 
>     * Combat Roll + QKL : https://mclo.gs/2I76WEa (NO Simply Skills!)
> 
> 
> ### Error message:
> 
> ```
> [11:25:48] [Render thread/ERROR]: Unreported exception thrown!
> java.lang.IllegalArgumentException: Can't find attribute combatroll:count
> 	at net.minecraft.class_5132.method_26865(class_5132.java:22) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.minecraft.class_5132.method_26862(class_5132.java:28) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.minecraft.class_5131.method_26852(class_5131.java:76) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.minecraft.class_1309.method_26825(class_1309.java:1904) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.combatroll.api.EntityAttributes_CombatRoll.getAttributeValue(EntityAttributes_CombatRoll.java:52) ~[transformed-mod-combatroll.i0:0/:?]
> 	at net.combatroll.client.RollManager.tick(RollManager.java:55) ~[transformed-mod-combatroll.i0:0/:?]
> 	at net.minecraft.class_310.tryRolling(class_310.java:3560) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.minecraft.class_310.handler$zpa000$combatroll$tick_TAIL(class_310.java:3552) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.minecraft.class_310.method_1574(class_310.java:1957) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.minecraft.class_310.method_1523(class_310.java:1181) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.minecraft.class_310.method_1514(class_310.java:802) ~[transformed-mod-minecraft.i0:0/:?]
> 	at net.minecraft.client.main.Main.main(Main.java:250) ~[minecraft-1.20.1-client.jar:?]
> 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
> 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
> 	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
> 	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
> 	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:551) ~[quilt-loader-0.21.2.jar:?]
> 	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:84) ~[quilt-loader-0.21.2.jar:?]
> 	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:28) ~[quilt-loader-0.21.2.jar:?]
> 	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87) ~[NewLaunch.jar:?]
> 	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130) ~[NewLaunch.jar:?]
> 	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
> ```

Thank you very much for your reply. I mainly posted this to let the developers know. I will have a look at the kotlin language mod. I mainly use Quilt for one single mod that switched from fabric to quilt so I was forced to install that otherwise I would have sticked to fabric. Thank you for your work and helping me narrow down the different causes, if it is a waste of time to fix this issue on the next version then consider this issue resolved. Have a wonderful day and I hope you guys learned something at least :)

**Comment by FedUpWizard** [comment]
Alright it seems I didn't even need the kotlin libraries installed for all my mods to work. I think I will close this issue then as resolved. Thanks again for your help 👍
